src/Components/Widgets/widgetDefaults.tsx

```diff
@@ -23,6 +23,11 @@ export const widgetMinHeight: { [widgetName: string]: number } = {
   ['notificationsEvents']: 1,
 };
 
+export const widgetHeaderLink: { [widgetName: string]: { title: string; href: string } | undefined } = {
```

Comment:
This is going to be problematic as we will have to keep in sync with both the backend and frontend. All metadata about all widgets should be mart of the backend which is supposed to be the source of truth.

The UI is "dumb" and its only doing what is coming from the backend. Additional parameters to the widgets would be done here: https://github.com/RedHatInsights/chrome-service-backend/blob/main/rest/service/dashboardTemplate.go#L14

Reply:
cc @karelhala @florkbr